database/src/rocks_db.rs

```diff
@@ -70,24 +70,24 @@ impl Database for RocksDB {
     }
 
     fn flush(&self) {
-        debug!("Flushing rocks db at {:?}", self.path);
+        log::debug!("Flushing rocks db at {:?}", self.path);
```

Comment:
Ok this is inconsistent, before I realised we were on 2018, will remove the namespace